### ISSUE TITLE
Action Pad and Action - Ensure implicit list markup is avoided - (2036569768)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7778]
### Violation: 
 Ensure implicit list markup is avoided 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) 

### Severity: 
 6 

### Media Type: 
 Lists 

--- 
### Areas for Remediation: 
- Title: Action Pad and Action - Ensure implicit list markup is avoided - (2036569768) 
- Module: Action Pad and Action 
**Issue**
There is content that appears as a list but does not use list markup. Examples include:

- Undo action, for display purposes only
- Redo action, for display purposes only
- Expand

**User Impact**
Screen reader users will have difficulty understanding that this content is a list.

**Code Reference** 
 ```
<calcite-action-pad layout="vertical" calcite-hydrated="">
  <calcite-action text="Undo" icon="undo" label="Undo action, for display purposes only" appearance="solid" scale="m" calcite-hydrated=""></calcite-action>
  <calcite-action text="Redo" icon="redo" label="Redo action, for display purposes only" appearance="solid" scale="m" calcite-hydrated=""></calcite-action>
  <calcite-action appearance="solid" scale="m" calcite-hydrated=""></calcite-action>
</calcite-action-pad>
 ``` 
**Suggestion**
Ensure implicit list markup is avoided. Lists must use ``` <ul> ```, ``` <ol> ```, ``` <li> ```, ``` <dl> ```, ``` <dt> ```, and ``` <dd> ``` elements as appropriate.

**Compliant Code Example** 
 ```
<ul layout="vertical" calcite-hydrated="">
  <li>
    <calcite-action text="Undo" icon="undo" label="Undo action, for display purposes only" appearance="solid" scale="m" calcite-hydrated=""></calcite-action>
  </li>
  <li>
    <calcite-action text="Redo" icon="redo" label="Redo action, for display purposes only" appearance="solid" scale="m" calcite-hydrated=""></calcite-action>
  </li>
  <li>
    <calcite-action appearance="solid" scale="m" calcite-hydrated=""></calcite-action>
  </li>
</ul>
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
This issue doesn't seem actionable.

**Comment by macandcheese** [comment]
Propose closing as a result of discussion in https://github.com/Esri/calcite-design-system/issues/7052. These really should behave more as toolbars.

**Comment by geospatialem** [comment]
Closing per https://github.com/Esri/calcite-design-system/issues/7778#issuecomment-1728284438 and https://github.com/Esri/calcite-design-system/issues/7778#issuecomment-1799536807. Refer to the repurposed https://github.com/Esri/calcite-design-system/issues/7052.
